terraform/load-balancing.tf

```diff
@@ -43,6 +43,37 @@ resource "aws_lb_listener" "validator-ac" {
   }
 }
 
+resource "aws_lb" "validator-jsonrpc" {
+  name                             = "${terraform.workspace}-jsonrpc"
+  load_balancer_type               = "network"
+  enable_cross_zone_load_balancing = true
+  subnets                          = aws_subnet.testnet.*.id
+}
+
+resource "aws_lb_target_group" "validator-jsonrpc" {
+  name     = "${terraform.workspace}-jsonrpc"
+  protocol = "TCP"
+  port     = 5000
```

Comment:
I'd prefer something like 8080 for the internal port, since it's HTTP.

Reply:
This is the number they pick in https://github.com/libra/libra/pull/2861

Reply:
yes, but we're ok to change default port to whatever makes more sense for you guys